### PR TITLE
GP2-922 Country selector tweaks

### PR DESCRIPTION
**Pull request by bobmeredith**
- [x] Ticket exists in Jira  https://uktrade.atlassian.net/browse/GP2-922
 - [x] Jira ticket has the correct status.
 - [x] [Changelog](CHANGELOG.md) entry added.

 - [x] (if is a UI change) Includes screenshot(s) - ideally before and after, but at least after
 
 - [X] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

**Comment by bobmeredith** [comment]
image

**Comment by bobmeredith** [comment]
image

File: react-components/src/components/ProductFinder/CountryFinder.jsx [hunk 1]
```diff
@@ -58,9 +60,9 @@ export function CountryFinderModal(props) {
     setScrollShadow()
   }
 
-  const searchChange = (evt) => {
-    setExpandRegion(evt.target.value.length > 0)
-    setSearchStr(evt.target.value.toUpperCase())
+  const searchChange = (value) => {
```
Comment (dahfool) [inline react-components/src/components/ProductFinder/CountryFinder.jsx:63]: @bobmeredith Do you need yo use `target` here ?

File: react-components/src/components/ProductFinder/Interaction.jsx [hunk 2]
```diff
@@ -37,8 +37,9 @@ function RadioButtons(props) {
           name={attribute.id}
           value={option.id}
           data-label={option.name}
-          defaultChecked={checked}
+          checked={checked}
           aria-label={option.name}
+          onChange={changeVal}
         />
         {/* eslint-disable jsx-a11y/label-has-associated-control */}
```
Comment (dahfool) [inline react-components/src/components/ProductFinder/Interaction.jsx:44]: Might be good to remove -   {/* eslint-disable jsx-a11y/label-has-associated-control */}

File: react-components/src/components/ProductFinder/CountryFinder.jsx (hunk 1 above)
Comment (bobmeredith) [reply react-components/src/components/ProductFinder/CountryFinder.jsx:63]: This used to be an event handler, but now the event is handled within the SearchInput component and the value is passed as the argument to the searchChange callback.

**Review by dahfool — APPROVED** [review]

**Pull request merged**